cpu/stm32/stmclk/stmclk_l4wx.c

```diff
@@ -643,5 +643,14 @@ void stmclk_init_sysclk(void)
         gpio_init_af(GPIO_PIN(PORT_A, 8), GPIO_AF0);
     }
 
+#if IS_USED(MODULE_PERIPH_USBDEV)
+    /* USBDEV is clocked by HSI48 so enable this clock when the peripheral is used */
+    RCC->APB1RSTR1 |= RCC_APB1RSTR1_USBRST;
```

Comment:
Has the reset to be done before enabling the 48Mhz clock or could it be done after ?

Reply:
Dunno to be honest, but I can give it a try.

Reply:
Move reset after 48MHz clock enable.